### PR TITLE
Added daily integration tests for css and qss submit_qubo

### PR DESCRIPTION
**Pull request by pnoell**

### CODE REVIEW

**Comment by pnoell** [comment]
Fixes #700

File: cirq-superstaq/cirq_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -282,3 +283,13 @@ def test_submit_to_provider_simulators(target: str, service: css.Service) -> Non
 
     job = service.create_job(circuit=circuit, repetitions=1, target=target)
     assert job.counts() == {"11": 1}
+
+
+def test_submit_qubo(provider: css.Service) -> None:
```
Comment (vtomole) [inline cirq-superstaq/cirq_superstaq/daily_integration_test.py:288]: Nit
```suggestion
def test_submit_qubo(service: css.Service) -> None:
```

File: cirq-superstaq/cirq_superstaq/daily_integration_test.py [hunk 2]
```diff
@@ -282,3 +283,13 @@ def test_submit_to_provider_simulators(target: str, service: css.Service) -> Non
...
+    test_qubo = {(0,): -1, (1,): -1, (2,): -1, (0, 1): 2, (1, 2): 2}
+    serialized_result = provider.submit_qubo(
```
Comment (vtomole) [inline cirq-superstaq/cirq_superstaq/daily_integration_test.py:290]:
```suggestion
    serialized_result = service.submit_qubo(
```

**Review by vtomole — APPROVED** [review]
LGTM. [I've confirmed that this works ](https://github.com/Infleqtion/client-superstaq/actions/runs/5905521628)

**Pull request merged**